frontend/src/pages/home.tsx

```diff
@@ -113,19 +113,22 @@ const Home: FC = () => {
                 </TabList>
               )}
               {extraSmall && (
-                <div className="px-4 py-2">
+                <div className="">
```

Comment:
Use React.Fragment `<></>` instead of Div

Reply:
Fixed